.github/workflows/ci.yml

```diff
@@ -10,14 +10,14 @@ on:
 
 env:
   # Common versions
-  GO_VERSION: '1.19'
+  GO_VERSION: '1.19.4'
   GOLANGCI_VERSION: 'v1.49'
   DOCKER_BUILDX_VERSION: 'v0.8.2'
 
   # Common users. We can't run a step 'if secrets.AWS_USR != ""' but we can run
   # a step 'if env.AWS_USR' != ""', so we copy these to succinctly test whether
   # credentials have been provided before trying to run steps that need them.
-  DOCKER_USR: ${{ secrets.DOCKER_USR }}
+  UPBOUND_ROBOT_ID: ${{ secrets.UPBOUND_ROBOT_ID }}
```

Comment:
Not picky about names, but perhaps for consistency other repos publishing to Marketplace use `XPKG_ACCESS_ID` and `XPKG_TOKEN` , e.g. [provider-aws](https://github.com/crossplane-contrib/provider-aws/blob/master/.github/workflows/ci.yml#L309-L311)

Reply:
good suggestion, done.

Reply:
Hmm, on second thought these aren't going to be xpkgs by definition, so that wouldn't be an accurate name.

Reply:
But it is interacting with the Marketplace OCI URL which is xpkg.upbound.io so it's at least half appropriate.

Reply:
Fair point 👍